### PR TITLE
Add SingleList to Home component and ListItem to List component

### PR DESCRIPTION
**Pull request by marshjaja**
## Description

We rendered the shopping list the users have access to. And the name of the items on the list.

## Related Issue
 Issue #2 

## Acceptance Criteria

-  In Home.jsx, the SingleList component is used to render the name of each shopping list a user is subscribed to
 - In List.jsx, the ListItem component is used to render the name of each item

## Type of Changes

Added The Shopping list feature.

### Before

N/A

### After

Items displayed

## Testing Steps / QA Criteria

- Do a `git pull` and `git checkout  mm-jo-rendered-list`.
- Open the homepage by running `npm start`.
- Shopping will render on Home component and list items will show in List component.

### CODE REVIEW

File: src/views/Home.jsx [hunk 1]
```diff
@@ -7,10 +8,17 @@ export function Home({ data, setListPath }) {
 				Hello from the home (<code>/</code>) page!
 			</p>
 			<ul>
-				{/**
-				 * TODO: write some JavaScript that renders the `lists` array
-				 * so we can see which lists the user has access to.
-				 */}
+				{/* {data.map((item) => (
+					<SingleList key={item.id} name={item.name} path={item.path} setListPath={setListPath} />
+				))} */}
```
Comment (Wyna-7) [inline src/views/Home.jsx]: If this is not needed, it is best to remove commented out code :)

**Review by Wyna-7 — CHANGES_REQUESTED** [review]
Great work!

**Review by Hudamabkhoot — APPROVED** [review]
Well done!

**Review by hariscs — APPROVED** [review]
Good work guys  🎉

**Pull request merged**